cmd/qosserver/main_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var server, client *net.UDPConn
+var err error
+
+var _ = Describe("QoS Server tests", func() {
+	BeforeEach(func() {
+		server, err = createServer(3070)
+		Expect(err).To(Succeed())
+		client, err = createServer(3071)
+		Expect(err).To(Succeed())
+	})
+	AfterEach(func(){
+		err = server.Close()
+		Expect(err).To(Succeed())
+		err = client.Close()
+		Expect(err).To(Succeed())
+	})
+	It("testing the udp server sends the response to the sender", func() {
```

Comment:
can you write the messages like It("should respond to the sender") etc.? 
We probably need to fix these message throughout the project, sorry about that